docs-syntax-guide.rst

```diff
@@ -0,0 +1,1377 @@
+
+.. _section-labels:
+
+Section labels
+~~~~~~~~~~~~~~~~
+
+In order to facilitate efficient :ref:`cross-referencing`, sections should be labeled. This is done on the line above the section title. The format is:
+
+- two dots
+- underscore
+- section label
+
+  - lowercase
+  - hyphen separators
+
+- a single colon
+
+.. code-block:: rst
+
+  .. _section-label:
+
+  Section Title
+  ----------------
+
+  Lorem ipsum content of section blah blah.
+
+The section label is usually a slugified version of the section title.
+
+Section titles must be unique throughout the entire documentation set. Therefore, if you write a common title that might appear in more than one document (*Learn More* or *Getting Started*, for example), you'll need to include additional words to make the label unique. The best way to do this is to add a meaningful work from the document title.
+
+.. code-block:: rst
+
+  ODK Aggregate
+  ===============
+
+  ODK Aggregate is a server application...
+
+  .. _aggregate-getting-started:
+
+  Getting Started
+  -----------------
+
+.. _basic-markup:
+
+Basic Markup
+-------------
+
+.. _escaping-characters:
+
+Escaping characters
+~~~~~~~~~~~~~~~~~~~~~
+
+Markup characters can be escaped using the ``\`` character.
+
+.. code-block:: rst
+
+  *Italic.*
+
+  \*Not italic, surrounded by asterisks.\*
+
+*Italic.*
+
+\*Not italic, surrounded by asterisks.\*
+
+.. _inline-markup:
+
+Emphasis and Inline Literal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: rst
+
+  Single asterisks for *italic text* (``<em>``).
+
+  Double asterisks for **bold text** (``<strong>``).
+
+  Double back-ticks for ``inline literal text`` (``<code>``).
+
+
+Single asterisks for *italic text* ( ``<em>`` ).
+
+Double asterisks for **bold text** ( ``<strong>`` ).
+
+Double back-ticks for ``inline literal text`` ( ``<code>`` ).
+
+.. note::
+
+  The **bold**, *italic*, and ``inline literal`` styles do not carry semantic meaning. They should not be used when a more semantically appropriate markup construct is available; for example, when :ref:`writing about GUI text <interface-writing>`.
+
+
+.. _hyperlinks:
+
+Hyperlinks
+~~~~~~~~~~~~
+
+**External** hyperlinks — that is, links to resources *outside* the documentation — look like this:
+
+.. code-block:: rst
+
+  This is a link to `example <http://example.com>`_.
+
+This is a link to `example <http://example.com>`_.
+
+You can also use "reference style" links:
+
+.. code-block:: rst
+
+  This is a link to `example`_.
+
+  .. _example: http://example.com
+
+This may help make paragraphs with *a lot* of links more readable. In general, the inline style is preferable. If you use the reference style, be sure to keep the link references below the paragraph where they appear.
+
+.. code-block:: rst
+
+  You can also simply place an unadorned URI in the text: http://example.com
+
+You can also simply place an unadorned URI in the text: http://example.com
```

Comment:
I think it would be more appropriate to use URL instead of URI here due to the inclusion of the “access mechanism”, that is "http://"